coral/src/domain/topic/topic-test-helper.test.ts

```diff
@@ -33,7 +33,7 @@ describe("topic-test-helper.ts", () => {
   });
 
   describe("createMockTopicApiResponse", () => {
-    it("creates a mocked api response for topics with 2 entries with current page 1 per default", () => {
+    it("creates a mocked api response for browse-topics with 2 entries with current page 1 per default", () => {
```

Comment:
I assume the domain should not know about specific app features. This comment applies to all changes in this file